android/src/com/mapswithme/maps/MapFragment.java

```diff
@@ -152,7 +152,8 @@ public void surfaceCreated(SurfaceHolder surfaceHolder)
     getActivity().getWindowManager().getDefaultDisplay().getMetrics(metrics);
     final float exactDensityDpi = metrics.densityDpi;
 
-    mEngineCreated = nativeCreateEngine(surface, (int) exactDensityDpi);
+    boolean isFirstLaunch = false; //TODO(Android team): set correct value
```

Comment:
А если в момент экрана приветственного старта приложение было свернуто, а затем развернуто?

> On 14.4.2016, at 11.33, Roman Kuznetsov <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/MapFragment.java:
> 
> > @@ -152,7 +152,8 @@ public void surfaceCreated(SurfaceHolder surfaceHolder)
> >      getActivity().getWindowManager().getDefaultDisplay().getMetrics(metrics);
> >      final float exactDensityDpi = metrics.densityDpi;
> >  
> > -    mEngineCreated = nativeCreateEngine(surface, (int) exactDensityDpi);
> > +    boolean isFirstLaunch = false; //TODO(Android team): set correct value
> Она должна быть true, когда показывается приветственный экран первого старта.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>